synapse/logging/opentracing.py

```diff
@@ -466,8 +466,16 @@ def init_tracer(hs: "HomeServer") -> None:
         STRIP_INSTANCE_NUMBER_SUFFIX_REGEX, "", hs.get_instance_name()
     )
 
+    jaeger_config = hs.config.tracing.jaeger_config
+    tags = jaeger_config.setdefault("tags", {})
+
+    # tag the Synapse instance name so that it's an easy jumping
+    # off point into the logs. Can also be used to filter for an
+    # instance that is under load.
+    tags.setdefault(SynapseTags.INSTANCE_NAME, hs.get_instance_name())
```

Comment:
Are you using `setdefault` to not stomp a value that's already there?

```suggestion
    tags.setdefault(SynapseTags.INSTANCE_NAME, hs.get_instance_name())
```

Reply:
Yeah. It's probably overkill for the tag itself. I'll just use the normal index notation.

Reply:
```suggestion
    tags[SynapseTags.INSTANCE_NAME] = hs.get_instance_name()
````